consensus/src/epoch_manager.rs

```diff
@@ -1221,7 +1223,7 @@ impl<P: OnChainConfigProvider> EpochManager<P> {
         fast_rand_config: Option<RandConfig>,
         rand_msg_rx: aptos_channel::Receiver<AccountAddress, IncomingRandGenRequest>,
     ) {
-        match self.storage.start() {
+        match self.storage.start(None) {
```

Comment:
this doesn't work, it breaks the fault tolerance if order vote is enabled

Reply:
Why does it break fault tolerance?

Reply:
node restarts with None would panic when it can't find root without the corresponding order cert